Automattic/newspack-plugin
# Update design campaigns wizard

### All Submissions:






### Changes proposed in this Pull Request:

This PR adds a bunch of small tweaks to the Campaigns Wizard

* Select control: Add support to help text
* Campaigns: Update the preview icon
* Campaigns: Fix links colour in segments header
* Segments: Fix margin between draggable elements
* Settings: Use `Grid` to reorganise content

### How to test the changes in this Pull Request:

1. Visit the Campaigns Wizard -- check all the different tabs
2. Switch to this branch
3. Check all the different tabs again and notice the slight differences

### Other information:
